apps/main/src/dex/components/PageRouterSwap/utils.ts

```diff
@@ -65,7 +65,7 @@ export function parseRouterRoutes(
         try {
           pool = getPool(route.poolId)
         } catch (error) {
-          log('routerBestRouteAndOutput missing poolName', route)
+          log('routerBestRouteAndOutput missing poolName', LogStatus.ERROR, route.poolId)
```

Comment:
Not important but I'd be in favor of removing specific function variants like `logSuccess` for the more generic variant with logstatus parameter like `log("foo", LogStatus.SUCCESS)

Or instead of this variant use `logError`. Either way pick a side 😄

Reply:
Makes sense. The `logSuccess` and `logError` functions expect a query key though (we could change that)

Reply:
I merged now, but we can improve separately